enhancements/evidence-forwarding.md

```diff
@@ -0,0 +1,320 @@
+   enhancement, explain why but do not remove the section. This part
+   of the process is enforced by the linter CI job.
+
+See ../README.md for background behind these instructions.
+
+Start by filling out the header with the metadata for this enhancement.
+
+# Evidence Forwarding
+
+## Summary
+
+Compliance audits usually require evidence that proves an environment is
+compliant. While the compliance operator provides check results as a Custom
+Resource, users an generate complete reports in an [Asset Reporting Format
+(ARF)](https://csrc.nist.gov/Projects/Security-Content-Automation-Protocol/Specifications/arf).
+These reports can be large and verbose. A single report for a small cluster can
+generate about 3 MB of data. This increase with each scan run, and with the
+amount of infrastructure being scanned.
+
+Historically, the Compliance Operator saves these reports to a physical volume.
+This works around saving the entire report in Custom Resource Definitions,
+which are bound to limitations of etcd.
+
+Unfortunately, physical volumes have their own short-comings. Reports stored in
+physical volumes still need to be fetched for evidence. Physical volumes can
+also be deleted like any other cluster resource.
+
+A improvement to the idea of putting evidence reports on disk would be to dump
+them into an object or file store. This has an added benefit of allowing
+administrators managing multiple clusters to put evidence in a single location.
+
+## Motivation
+
+The motivation of the enhancement is to store compliance evidence outside the
+cluster and its storage pools.
+
+### Goals
+
+The goal is to provide a way for Kubernetes administrators to configure the
+Compliance Operator to send compliance reports, specifically ARF reports, to an
+external storage pool, like an object or file store.
+
+### Non-Goals
+
+It is not the goal of this enhancement to implement results forwarding, or
+adjust the definition of evidence outside an ARF report.
+
+## Proposal
+
+This is where we get down to the nitty gritty of what the proposal
+actually is. Describe clearly what will be changed, including all of
+the components that need to be modified and how they will be
+different. Include the reason for each choice in the design and
+implementation that is proposed here, and expand on reasons for not
+choosing alternatives in the Alternatives section at the end of the
+document.
+
+### User Stories
+
+1. As a Kubernetes administrator, I want to configure the Compliance Operator
+   to send all ARF reports to a dedicated object store.
+2. As a Kubernetes administrator, I want to opt out of storing any ARF reports
+   on physical volumes.
+3. As a Kubernetes administrator, I expect verbose alerts when the Compliance
+   Operator cannot send ARF reports to a configured external storage pool.
+
+### API Extensions
+
+API Extensions are CRDs, admission and conversion webhooks, aggregated API servers,
```

Comment:
Add a section here that describes the configuration changes to the CRDs that would be required to fill the use cases above.